ci-operator/step-registry/cucushift/installer/rehearse/gcp/upi/baselinecaps/vcurrent/provision/cucushift-installer-rehearse-gcp-upi-baselinecaps-vcurrent-provision-chain.yaml

```diff
@@ -0,0 +1,20 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-upi-baselinecaps-vcurrent-provision-chain
+  steps:
+  - ref: ipi-install-rbac
+  - chain: upi-conf-gcp
+  - ref: upi-install-gcp
+  - ref: ipi-install-times-collection
+  - ref: enable-qe-catalogsource
+  env:
+    - name: BASELINE_CAPABILITY_SET
+      default: "vCurrent"
```

Comment:
@jinyunma @shellyyang1989 I remember the default value of baselineCapabilitySet is "vCurrent", right? If so, I'm thinking whether we still need to test the default value in prow, wdyt?

Reply:
yes, default baselineCapabilitySet is vCurrent. The purpose of adding this workflow is just to make sure that installer can set correct value and cvo could parse that. If no need to test in Prow CI, I can remove it. Anyway, we already cover that in installer test cases.

Reply:
if installer cases already cover it, I guess we can drop it in prow ci.

Reply:
ok, will drop this workflow, and move workflow cucushift-installer-rehearse-azure-upi-baselinecaps-v411 onto gcp platform as azure have limited slice quota.

Reply:
@yunjiang29 here is the discussion about `vCurrent`. As default baselineCapabilitySet is vCurent, we plan to cover it only in our test cases.

Reply:
@jinyunma get it, thanks.